airbyte-webapp/src/packages/cloud/views/workspaces/WorkspacesPage/components/CreateWorkspaceForm.tsx

```diff
@@ -49,7 +49,7 @@ const CreateWorkspaceForm: React.FC<CreateWorkspaceFormProps> = ({ onSubmit }) =
         <CreationForm>
           <Field name="name">
             {({ field, meta }: FieldProps<string>) => (
-              <ClearInput {...field} autoFocus type="text" error={!!meta.error && meta.touched} />
```

Comment:
did not appear to make any functional difference to remove this.  autofocus still happens.